package.json

```diff
@@ -37,6 +37,7 @@
     "dom"
   ],
   "dependencies": {
+    "domhandler": "^4.0.0",
```

Comment:
Do you mind making this an exact dependency? `4.0.0`

Reply:
not at all